### ISSUE TITLE
Market: Add multitests for sudo messages

### ISSUE THREAD

**Issue by ueco-jb** [osmosis-labs/isotonic#93]
I'm currently unable to test sudo messages in multitest without bindings, which will be added in another task.
I'll create an issue as follow-up to add proper tests after bindings are done.

_Originally posted by @ueco-jb in https://github.com/confio/isotonic/issues/90#issuecomment-1058125395_

Blocked by https://github.com/confio/isotonic/issues/84
